src/main/g8/src/main/scala/$package$/infrastructure/$if(add_caliban_endpoint.truthy || add_server_sent_events_endpoint.truthy || add_websocket_endpoint.truthy)$EventSubscriber.scala$endif$

```diff
@@ -0,0 +1,39 @@
+package $package$.infrastructure
+import $package$.domain.{Subscriber, ItemId }
+import zio._
+import zio.logging._
+import zio.stream.ZStream
+
+class EventSubscriber(env: Logging, deletedEventsSubscribers: Ref[List[Queue[ItemId]]]) extends Subscriber.Service {
```

Comment:
I believ `env: Logging` is not needed here.

Reply:
Are you sure? In publishing, we have logging 
```Scala
def publishDeleteEvents(deletedItemId: ItemId): ZIO[Any, Nothing, List[Boolean]] =
    log.info(s"Publishing delete event for item \${deletedItemId.value}").provide(env) *>
      deletedEventsSubscribers.get.flatMap[Any, Nothing, List[Boolean]] { subs =>
        // send item to all subscribers
        log.info("subs" + subs)
        UIO.foreach(subs) { queue =>
          log.info("queue" + queue)```